pkg/api/vm/handler.go

```diff
@@ -47,6 +48,7 @@ type vmActionHandler struct {
 	restores                  ctlharvesterv1.VirtualMachineRestoreClient
 	settingCache              ctlharvesterv1.SettingCache
 	nodeCache                 ctlcorev1.NodeCache
+	dataVolumesCache          ctlcdiv1beta1.DataVolumeCache
```

Comment:
Let's coordinate with the DataVolume changes on the latest master

Reply:
Updated. Thanks